src/components/button-share/link-icon.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
```

Comment:
Icons should live in the design system and be used like this:
`import ExpandMoreIcon from "@danskernesdigitalebibliotek/dpl-design-system/build/icons/collection/ExpandMore.svg";`

Reply:
How would you go around rendering an SVG without an ´img´ tag this way?